checked/debug.go

```diff
@@ -191,16 +229,13 @@ func getDebuggerRef(c *RefCount) *debuggerRef {
 
 func tracebackEvent(c *RefCount, ref int, e debuggerEvent) {
 	d := getDebuggerRef(c)
-	stack := debug.Stack()
-	endlines := 0
-	skipPrologue := 1
+	depth := tracebackMaxDepth
+	pc := tracebackCallersPool.Get().([]uintptr)
+	if capacity := cap(pc); capacity < depth {
```

Comment:
We could potentially reach this case if someone changes the traceback cycles during the runtime of the program.  Essentially just some defensive programming.